mllib/src/main/scala/org/apache/spark/ml/r/AFTSurvivalRegressionWrapper.scala

```diff
@@ -30,6 +32,7 @@ import org.apache.spark.ml.regression.{AFTSurvivalRegression, AFTSurvivalRegress
 import org.apache.spark.ml.util._
 import org.apache.spark.sql.{DataFrame, Dataset}
 
+@Since("2.0.0")
 private[r] class AFTSurvivalRegressionWrapper private (
```

Comment:
OK,
We have added the annotations already and the versions are very precise because we have developed a tool to find the exact version a class appeared in, for the first time.
Should we remove the annotations from private classes?